backend/src/main/java/com/staz/controller/UserController.java

```diff
@@ -0,0 +1,136 @@
+package com.staz.controller;
+
+
+import com.staz.service.UsersService;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.http.HttpStatus;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
```

Comment:
```suggestion
@RestController
@RequestMapping("/user")
```
Wtedy wszystkie metody w tym kontrolerze zmapują się na `/user`